Application/EdFi.Ods.AdminApi.AdminConsole/Features/Instances/ReadInstances.cs

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.AdminApi.AdminConsole.Features.OdsInstances;
+using EdFi.Ods.AdminApi.AdminConsole.Infrastructure.Services.Instances.Queries;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+
+namespace EdFi.Ods.AdminApi.AdminConsole.Features.UserProfiles;
+
+public class ReadInstances : IFeature
+{
+    public void MapEndpoints(IEndpointRouteBuilder endpoints)
+    {
+        AdminApiAdminConsoleEndpointBuilder.MapGet(endpoints, "/instances", GetInstances)
+            .BuildForVersions();
+
+        AdminApiAdminConsoleEndpointBuilder.MapGet(endpoints, "/instances/{id}", GetInstance)
+            .WithRouteOptions(b => b.WithResponse<InstanceModel>(200))
+            .BuildForVersions();
+    }
+
+    internal async Task<IResult> GetInstances([FromServices] IGetInstancesQuery getInstancesQuery)
+    {
+        var instances = await getInstancesQuery.Execute();
+        return Results.Ok(instances);
```

Comment:
I'm not sure if makes sense to return the entire payload, I would say and in terms to be consistent with adminconsole, we should return only the property "document" or a list of them, with this, it might help us to integrate it easily in the adminconsole webapp.

See the example
https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-2.x/blob/main/docs/design/adminconsole/notes.md#endpoints